### ISSUE TITLE
NvimTree change_root callback failure

### ISSUE THREAD

**Issue by Valinora** [LunarVim/LunarVim#3970]
### Problem description

When opening lazygit or toggleterm for the first time after starting lvim with NvimTree open, there is a callback error related to changing NvimTree's root directory. It doesn't _seem_ to be causing any issues, as NvimTree's root is being set correctly to the project folder, and neither lazygit or toggleterm are misbehaving. 

### LunarVim version

master-2e42dddb

### Neovim version (>= 0.8.0)

NVIM v0.8.3

### Operating system/version

6.2.7-arch1-1

### Steps to reproduce

In the latest pinned rolling release:
1. Open a project folder using either alpha.nvim start page, or any file in the project folder from the shell (e.g. ` lvim src/main.rs`)
2. Open NvimTree
3. Open either lazygit `<leader>gg` or toggleterm `<C-\>`
4. NvimTree has a cd callback error.

### Screenshots

```lua
Error executing vim.schedule lua callback: .../nvim-tree.lua/lua/nvim-tree/actions/root/change-dir.lua:50: Vim(lcd):E472: Command failed
stack traceback:
        [C]: in function 'cmd'
        .../nvim-tree.lua/lua/nvim-tree/actions/root/change-dir.lua:50: in function 'cd'
        .../nvim-tree.lua/lua/nvim-tree/actions/root/change-dir.lua:67: in function 'f'
        .../nvim-tree.lua/lua/nvim-tree/actions/root/change-dir.lua:60: in function 'force_dirchange'
        .../nvim-tree.lua/lua/nvim-tree/actions/root/change-dir.lua:46: in function 'fn'
        ...arvim/site/pack/lazy/opt/nvim-tree.lua/lua/nvim-tree.lua:74: in function 'change_root'
        ...t/nvim-tree.lua/lua/nvim-tree/actions/tree/find-file.lua:59: in function 'fn'
        ...arvim/site/pack/lazy/opt/nvim-tree.lua/lua/nvim-tree.lua:375: in function 'callback'
        ...site/pack/lazy/opt/nvim-tree.lua/lua/nvim-tree/utils.lua:336: in function <...site/pack/lazy/opt/nvim-tree.lua/lua/nvim-tree/utils.lua:335>
Press ENTER or type command to continue
```

**Comment by LostNeophyte** [comment]
thanks for filling out steps to reproduce thoroughly

the issue in nvim-tree: https://github.com/nvim-tree/nvim-tree.lua/issues/2057, it looks like there's already a pr that will fix it

**Comment by Valinora** [comment]
Looks like their fix has merged into master, should just need to update the pinned version here.

**Comment by Valinora** [comment]
Fixed by #3968
